core/src/main/kotlin/org/lflang/generator/ts/TSFileConfig.kt

```diff
@@ -53,4 +54,8 @@ class TSFileConfig(
         super.doClean()
         FileUtil.deleteDirectory(srcGenPath)
     }
+
+    override fun getExecutableExtension(): String {
+        return if (GeneratorUtils.isHostWindows()) ".bat" else ""
```

Comment:
Why would TS use `.bat` and other targets `.exe` (see FileConfig.getExecutableExtension()`).

Reply:
Python target also uses `.bat` in windows(see `PyFileConfig.java`). I think this is because C, C++ and Rust compiles code into binary, while python and typescript generates a script to run the generated code.

Reply:
:+1: